tests/metafake.h

```diff
@@ -0,0 +1,149 @@
+#ifndef METAFAKE_H_
+#define METAFAKE_H_
+
+#include <boost/filesystem.hpp>
+#include <chrono>
+#include <fstream>
+#include <string>
+
+#include "logging/logging.h"
+#include "utilities/utils.h"
+
+#define SHELL_CMD(cmd) if (system(cmd)) {LOG_ERROR << "system command failed: " << cmd;}
+
+class MetaFake {
+  public:
+  MetaFake(const boost::filesystem::path &meta_dir_in): meta_dir(meta_dir_in) {
+      // generate original repo data
+      cmd_head = "build/src/aktualizr_repo/aktualizr-repo --path " + meta_dir.string() + " --command ";
```

Comment:
this is breaking on ci, because it depends on the cmake build dir being named `build` and sitting in the repo's root.

It would maybe be nicer to use the native c++ api of aktualizr-repo: https://github.com/advancedtelematic/aktualizr/blob/master/src/aktualizr_repo/uptane_repo.h as it wouldn't require shelling out some processes.